### PR TITLE
added html error pages

### PR DESCRIPTION
**Pull request by michelle-mok**

### CODE REVIEW

File: soyuz_app/views/error.py [hunk 1]
```diff
@@ -3,17 +3,15 @@
...
 # HttpResponseNotAllowed
 # HttpResponseGone
-from django.http import HttpResponseNotFound, HttpResponseServerError
+from django.shortcuts import render
 from sentry_sdk import capture_message
 
 
-def error_404(*args, **kwargs):
-    capture_message("Page not found!", level="error")
+def custom_error_404(request, exception):
```
Comment (awongh) [inline soyuz_app/views/error.py:10]: why did you need to change the name of the views?

File: soyuz_project/urls.py [hunk 2]
```diff
@@ -21,11 +21,11 @@
...
 from django.urls import include, path
 
-from soyuz_app.views.error import error_404, error_500
+from soyuz_app.views.error import custom_error_404, custom_error_500
 
 # assign the custom error handling views
-handler404 = error_404  # noqa: F811
-handler500 = error_500  # noqa: F811
+handler404 = custom_error_404  # noqa: F811
+handler500 = custom_error_500  # noqa: F811
```
Comment (awongh) [inline soyuz_project/urls.py:28]: could you have left this file alone if you didn't change the name of the views?

File: soyuz_project/settings/local.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 # GENERAL
 # ------------------------------------------------------------------------------
 # https://docs.djangoproject.com/en/dev/ref/settings/#debug
-DEBUG = True
+DEBUG = False
```
Comment (awongh) [inline soyuz_project/settings/local.py:7]: I don't think you need to change this?

Comment (michelle-mok) [reply soyuz_project/settings/local.py:7]: i changed it because i wanted to see the error pages, or else it would show the django error page:
`You’re seeing this error because you have DEBUG = True in your Django settings file. Change that to False, and Django will display a standard 404 page.`
